streams/cluster/src/test/resources/register/happycase/core-site.xml

```diff
@@ -0,0 +1,134 @@
+  <configuration>
```

Comment:
OK agreed on this, and `correct_config` and `invalid_config` look good to me. Will address.